weave

```diff
@@ -1553,25 +1550,10 @@ EOF
     launch)
         deprecation_warnings "$@"
         check_not_running router $CONTAINER_NAME        $BASE_IMAGE
-        check_not_running proxy  $PROXY_CONTAINER_NAME  $BASE_EXEC_IMAGE
         check_not_running_plugin_v2
         COMMON_ARGS=$(common_launch_args "$@")
```

Comment:
Done in 2d665608b950 `TO SQUASH: Removed/simplified common_launch_args. Removed local as only supported by bash.`